syslog-ng/alpine.dockerfile

```diff
@@ -14,6 +18,9 @@ RUN apk add --update-cache \
 USER builder
 WORKDIR /home/builder
 ADD --chown=builder:builder apkbuild .
+
+RUN [ $DEBUG = false ] || (cd axoflow/syslog-ng && patch -p1 -i APKBUILD-debug.patch)
```

Comment:
you can use the `-d` option for `patch` to change directory

Reply:
Thanks, fixed!